en/docs/setup/migrating-preparing-for-migration.md

```diff
@@ -343,7 +343,7 @@ configurations are handled by the a single file named
 
 !!! warn "Note" 
     When you are migrating from older version of WSO2
-    Identity Server, you need to convert old configurations to new
+    Identity Server that 5.9.0, you need to convert old configurations to new
     configuration model as well.
 
```

Comment:
When migrating from an older WSO2 Identity Server version, make sure to convert the old configurations to the new configuration model.